shell/platform/embedder/platform_view_embedder.h

```diff
@@ -38,6 +39,7 @@ class PlatformViewEmbedder final : public PlatformView {
     VsyncWaiterEmbedder::VsyncCallback vsync_callback;  // optional
     ComputePlatformResolvedLocaleCallback
         compute_platform_resolved_locale_callback;
+    float display_refresh_rate;
```

Comment:
I decided not to plumb the callback all the way through to vsync waiter as currently `GetDisplayRefreshRate` is `const` and none of the platforms currently need to dynamically return the refresh rate of the display.

Though I believe we will need that at some point in the future.